### PR TITLE
reformat/update config-file.md

### PR DESCRIPTION
**Pull request by chrismetz09**

### CODE REVIEW

File: docs/plugins/db-plugins.md [hunk 1]
```diff
@@ -263,12 +263,12 @@ The Consul plugin provides access to a consul key-value data store.
 
 ## FileDB
 
-The fileDB plugin uses the file system of an operating system as a key-value data store. The filesystem plugin watches for pre-defined files or directories, reads a configuration and sends response events according to changes.
+The fileDB (filesystem) plugin uses the file system of an operating system as a KV data store. The filesystem plugin watches for pre-defined files or directories, reads a configuration file, and generates events according to configuration changes.
```
Comment (ondrej-fabry) [inline docs/plugins/db-plugins.md]: fileDB != filesystem

File: docs/plugins/db-plugins.md [hunk 2]
```diff
@@ -263,12 +263,12 @@ The Consul plugin provides access to a consul key-value data store.
...
 
-All configuration is resynced in the beginning (as for standard key-value data store). Configuration files then can be added, updated, moved, renamed or removed, plugin makes all of the necessary changes.
+All configuration data is resynced in the beginning just as it is for KV data stores. Configuration files then can be added, updated, moved, renamed or removed. The plugin performs all of the necessary changes.
 
 !!! danger "Important"
-    FileDB as datastore is read-only from the plugin perspective. Changes from within the plugin are not allowed.
+    The filesystem plugin treats the FileDB data store as `read-only`. Changes from within the plugin are not permitted.
```
Comment (ondrej-fabry) [inline docs/plugins/db-plugins.md]: Why not just keep it `FileDB` plugin? The naming is explicit for plugins any using different wording than is actually used in code might confuse someone and be harder to search for.

Comment (chrismetz09) [reply docs/plugins/db-plugins.md]: makes sense. changing

**Comment by ondrej-fabry** [comment]
> composite which is most often the case

Model prefix + Composition of some labels and field values
Example: Route

> specific containing a unique parameter (e.g. an interface name or a SPD

Model prefix + Value of single field
Example: Interface

> global allowing only one message of the given type to be stored under a global key.

Model prefix + Constant string 
Example: Nat44Global

**Pull request merged**